src/transformers/models/bert_japanese/tokenization_bert_japanese.py

```diff
@@ -70,7 +70,25 @@
 
 
 class BertJapaneseTokenizer(BertTokenizer):
-    """BERT tokenizer for Japanese text"""
+    r"""
+    Construct a BERT tokenizer for Japanese text, based on a MecabTokenizer.
+
+    Args:
+        vocab_file (:obj:`str`):
+            Path to a one-wordpiece-per-line vocabulary file.
+        do_lower_case (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether to lower case the input. Only has an effect when do_basic_tokenize=True.
+        do_word_tokenize (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether to do word tokenization.
+        do_subword_tokenize (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether to do subword tokenization.
+        word_tokenizer_type (:obj:`str`, `optional`, defaults to :obj:`"basic"`):
+            Type of word tokenizer.
+        subword_tokenizer_type (:obj:`str`, `optional`, defaults to :obj:`"wordpiece"`):
+            Type of subword tokenizer.
+        mecab_kwargs (:obj:`str`, `optional`, defaults to :obj:`None`):
```

Comment:
```suggestion
        mecab_kwargs (:obj:`str`, `optional`):
```
Nit: we don't write the default when it's None.